src/main/kotlin/com/dwolla/DwollaClient.kt

```diff
@@ -254,15 +255,23 @@ abstract class DwollaClient(@JvmField val environment: DwollaEnvironment) {
     }
 
     private fun makeRequest(request: Request): Response<String> {
-        val preparedRequest = prepareRequest(request)
-        val result = preparedRequest.responseString()
-        return handleResponse(result)
+        try {
```

Comment:
At the moment this PR swallows all `Exceptions` that occur making a request and rethrows them as `DwollaException`s including the original `Exception` as the `DwollaException`'s `cause` which is displayed in the stack trace.

We can evaluate adding more specific error handling (in this PR or another), but this at least makes it more clear the exception occurred making a request to Dwolla as opposed to somewhere else in their app.

Reply:
#520

Reply:
$superpowers/string/key/drakeo